data/110/890/541/3/1108905413.geojson

```diff
@@ -12,15 +12,15 @@
     "iso:country":"CA",
     "lbl:latitude":43.676352,
     "lbl:longitude":-79.628342,
-    "lbl:max_zoom":18.0,
+    "lbl:max_zoom":18,
```

Comment:
Same goes for `lbl` zooms, let's keep them float values (`18.0`, rather than `18`).